packages/plugin-vue/src/index.ts

```diff
@@ -137,6 +133,7 @@ export default function vuePlugin(rawOptions: Options = {}): Plugin {
       },
       set options(value) {
         options = value
+        ;({ filter, customElementFilter, refTransformFilter } = buildFilter())
```

Comment:
While this looks okay to me for this particular case, I think that this would be hard to read and maintain.
How about making `options` `reactive` and these filters `computed`?

Reply:
I'm not sure about introducing `@vue/reactivity` to the plugin. Increased bundle size and works almost samely.

Reply:
`vue` is already a peer dependency

Reply:
Done.